### PR TITLE
Add config file for XiC simulation

### PR DESCRIPTION
**Pull request by gluparel**
Config file for simulation of XiC to hadronic decays in pp collisions with Pythia 8.

### CODE REVIEW

File: MC/CustomGenerators/PWGHF/Pythia_pp_XiC.C [hunk 1]
```diff
@@ -0,0 +1,24 @@
+AliGenerator *GeneratorCustom(TString opt =)
```
Comment (miweberSMI) [inline MC/CustomGenerators/PWGHF/Pythia_pp_XiC.C]: I think, one needs to add a default argument here: String opt = ""

**Comment by miweberSMI** [comment]
https://github.com/alisw/AliDPG/pull/279/files#r303837536

**Comment by miweberSMI** [comment]
Dear all,

I tested the updated macro and it doesn't show any problems with running after modifying the first line as suggested (I did not check the results). Once I get green light from conveners (and the suggested modification), I will merge the PR.

Cheers,
Michael

**Comment by miweberSMI** [comment]
One more question to @gluparel: does one need the modification in AliGenPythia to trigger on anti-particles for this PR?

File: MC/CustomGenerators/PWGHF/Pythia_pp_XiC.C (hunk 1 above)
Comment (gluparel) [reply MC/CustomGenerators/PWGHF/Pythia_pp_XiC.C]: Hello, sorry I missed this message. The correction is done.

**Comment by miweberSMI** [comment]
Hi @gluparel,

great thanks. Merging now.

Cheers,
Michael

**Pull request merged**